app/assets/javascripts/filter.js.coffee

```diff
@@ -18,20 +18,20 @@ class Filter
 
   filter: ($el) ->
     value = $el.val()
-    if !$el.is(":checked")
+    unless $el.is(":checked")
       value = "!" + value
     @$("[data-filter-key=#{ @key }]").each (idx, el) ->
-      hidden = el.getAttribute("data-filter-value") != value
```

Comment:
I prefer this! Find it easier to spot a symbol than a word. I think because there's always a chance `isnt`is a var or something